src/script/calling/CallingRepository.test.ts

```diff
@@ -178,7 +201,7 @@ describe('CallingRepository', () => {
       const call = new Call(
         {domain: '', id: ''},
         createConversationId(),
-        undefined,
+        CONV_TYPE.CONFERENCE,
```

Comment:
just curious why all of these are changing to conference? any insight from your side

Reply:
fixes strict mode errors